metainfo.patch

```diff
@@ -0,0 +1,80 @@
+--- org.tigervnc.vncviewer.metainfo.xml.in	2023-01-31 21:29:23.779423487 +0000
++++ org.tigervnc.TigerVNC.metainfo.xml	2023-01-31 21:30:43.924526110 +0000
+@@ -7,12 +7,14 @@
+   notice are preserved. This file is offered as-is, without any warranty.
+ -->
+ <component type="desktop-application">
+-  <id>org.tigervnc.vncviewer</id>
++  <id>org.tigervnc.TigerVNC</id>
```

Comment:
The preference here would be that the upstream chosen app-id be used instead.

Reply:
Thanks, I'll change that now. I picked ".TigerVNC" to match the case of other apps on Flathub and to distinguish from other "vncviewer"s, but I suppose the reverse domain name does the latter.

Reply:
The name says `TigerVNC Viewer` so it's clear to the user